pkg/apis/kops/componentconfig.go

```diff
@@ -248,6 +248,8 @@ type KubeAPIServerConfig struct {
 	AuditLogMaxBackups *int32 `json:"auditLogMaxBackups,omitempty" flag:"audit-log-maxbackup"`
 	// The maximum size in megabytes of the audit log file before it gets rotated. Defaults to 100MB.
 	AuditLogMaxSize *int32 `json:"auditLogMaxSize,omitempty" flag:"audit-log-maxsize"`
+	// AuditPolicyFile is path to a audit policy configuraion file on the masters
+	AuditPolicyFile string `json:"auditPolicyFile,omitempty" flag:"audit-policy-file"`
```

Comment:
Fully qualified path and file name? It is not just the path.

/mnt/my-audit-file.comf